mantidproject/mantid
# New LoadWAND algorithm

Loads event nexus file, integrates events, sets wavelength, mask, and goniometer, and sets proton charge to monitor counts

**To test:**
Load the `HB2C_7000.nxs.h5` from unit test data or any WAND data file and see that the units are correct, event integrated, masking applied ...


**Release Notes** 
release notes added

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
